src/main/java/net/mcreator/element/types/Item.java

```diff
@@ -91,6 +92,13 @@
 	public Procedure onDroppedByPlayer;
 	public Procedure onFinishUsingItem;
 
+	// Ranged properties
+	public boolean enableRanged;
+	public boolean shootConstantly;
+	public ProjectileEntry projectile;
+	public Procedure onRangedItemUsed;
+	public Procedure useCondition;

```

Comment:
useCondition is too loose term as this explicitly refers to ranged use condition and not to general use condition. Please refactor all related to this, including translation and help tip key

Reply:
`shootCondition` would be a better name?

Reply:
I was thinking to `useRangedCondition`

Reply:
Or even better rangedUseCondition to follow how it is called in Dimension for portal

Reply:
Thanks for the quick fix!